.coveralls.yml

```diff
@@ -0,0 +1,4 @@
+service_name: travis-pro
+
```

Comment:
Don't impact the work but why a blank line?

Reply:
It's because I use vscode and in my settings has the rule:
`"files.insertFinalNewline": true`